src/stc/ScintillaWX.cpp

```diff
@@ -844,6 +845,31 @@ sptr_t ScintillaWX::WndProc(unsigned int iMessage, uptr_t wParam, sptr_t lParam)
       case SCI_GETDIRECTPOINTER:
             return reinterpret_cast<sptr_t>(this);
 
+#ifdef __WXMSW__
+      // ScintillaWin
+      case WM_IME_STARTCOMPOSITION: 	// dbcs
+          // TODO: We haven't ported WM_IME_COMPOSITION yet, so always use windowed IME to make it at least work
+          //if (KoreanIME() || imeInteraction == imeInline) {
+          if (false) {
+              return 0;
+          }
+          else {
```

Comment:
This is trivial, but `if (false)` is bound to provoke warnings from at least some compilers, so I'd rather remove this code entirely or at least comment out the entire `if/else` part.